lib/resources/dstu2/soarian_immunization.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: Immunization
+field_name_base_url: http://hl7.org/fhir/DSTU2/immunization-definitions.html#Immunization
+fields:
+
+- name: vaccineCode
+  required: 'Yes'
+  cardinality: 1..1
+  type: CodeableConcept
+  description: The vaccine that was administered or was to be administered.
+  binding:
+    description: Identifies the vaccine substance administered.
+    terminology:
+    - display: CVX
```

Comment:
yes. I will update terminology bindings to add “NDC” - https://www.hl7.org/fhir/ndc.html. Description – National Drug Code